pylint/checkers/refactoring/implicit_booleaness_checker.py

```diff
@@ -197,23 +196,32 @@ def _check_use_implicit_booleaness_not_comparison(
                         confidence=HIGH,
                     )
 
+    def _get_node_description(self, node: nodes.NodeNG) -> str:
+        return {
+            nodes.List: "list",
+            nodes.Tuple: "tuple",
+            nodes.Dict: "dict",
+            nodes.Const: "str",
+        }.get(type(node), "iterable")
+
     def _implicit_booleaness_message_args(
         self, literal_node: nodes.NodeNG, operator: str, target_node: nodes.NodeNG
-    ) -> tuple[str, str]:
+    ) -> tuple[str, str, str]:
         """Helper to get the right message for "use-implicit-booleaness-not-comparison"."""
-        collection_literal = "{}"
-        if isinstance(literal_node, nodes.List):
-            collection_literal = "[]"
-        if isinstance(literal_node, nodes.Tuple):
-            collection_literal = "()"
+        description = self._get_node_description(literal_node)
+        collection_literal = {
+            "list": "[]",
+            "tuple": "()",
+            "dict": "{}",
+        }.get(description, "iterable")
```

Comment:
I wonder would it be best to factor this logic out and into the `_get_node_description` method? It is only used in this one place.

Reply:
You're right, it does not make a lot of sense right now but I'm going to use it use-implicit-booleaness-not-len later :)